Examples/Swift/CustomViewController.swift

```diff
@@ -114,4 +118,53 @@ class CustomViewController: UIViewController, MGLMapViewDelegate {
     @IBAction func showFeedback(_ sender: Any) {
         present(feedbackViewController, animated: true, completion: nil)
     }
+    
+    func toggleStepsList() {
+        if let controller = stepsViewController {
+            controller.dismiss()
+            stepsViewController = nil
+        } else {
+            guard let routeController = routeController else { return }
+            
+            let controller = StepsViewController(routeProgress: routeController.routeProgress)
+            controller.delegate = self
+            addChildViewController(controller)
+            view.addSubview(controller.view)
+            
+            controller.view.topAnchor.constraint(equalTo: instructionsBannerView.bottomAnchor).isActive = true
+            controller.view.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
+            controller.view.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
+            controller.view.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
```

Comment:
I'm not a fan of how much boilerplate is necessary to make the slide action happen. @frederoni is there anything we can do to encapsulate this better?

Reply:
You could make `UIView.pinInSuperview()` public and replace these 4 lines w/
`controller.view.pinInSuperview()`. But pinInSuperview doesn't respect RTL so make sure you change `left->leading` and `right->trailing` in `UIView.pinInSuperview()` if you decide to go that way.